pkgs/development/tools/rust/cargo-risczero/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchCrate
+, rustPlatform
+, pkg-config
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-risczero";
+  version = "0.14.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-uZz0jJ3klaOrqzJ0BUVDHxl7lv6vt0GT6RgQuJeyeyk=";
+  };
+
+  cargoSha256 = "sha256-t++3+Ijn1ykjMcMsdoe/1xfaji+DQvhyiFe6M/Bpbt0=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+  ];
```

Comment:
```suggestion
  ] ++ lib.optionals stdenv.isDarwin [
    darwin.apple_sdk.frameworks.Security
  ];
```

Reply:
Thanks, I left out darwin cause I didn't have a convenient way to test it, but I actually have access to a Macbook now so I'll make sure this works on mac and update

Reply:
It is totally fine if we just get it to compile or mark it broken for darwin.

Reply:
Added this and `darwin` to the package inputs, builds and seems to work on mac x86 now.